index.bs

```diff
@@ -201,46 +201,52 @@ spec: webidl
+        of {{"granted"}}, {{"prompt"}}, or {{"denied"}}, indicating respectively
+        if the calling algorithm should:
+      </p>
 
-    <ul>
-      <li>succeed without prompting the user ({{"granted"}}),</li>
-      <li>show the user a prompt to decide whether to succeed ({{"prompt"}}),</li>
-      <li>or fail without prompting the user ({{"denied"}}).</li>
-    </ul>
+      <ul>
+        <li>succeed without prompting the user ({{"granted"}}),</li>
+        <li>show the user a prompt to decide whether to succeed ({{"prompt"}}),</li>
+        <li>or fail without prompting the user ({{"denied"}}).</li>
+      </ul>
 
-    <p>
-      The UA returns whichever of these values most accurately reflects the user's
-      intent. Subsequent uses of |descriptor|'s
-      <a>permission state</a> with the same <a>current settings object</a>
-      must return the same value, unless the UA receives <a>new information
-      about the user's intent</a>.
-    </p>
+      <p>
+        The UA returns whichever of these values most accurately reflects the user's
+        intent. Subsequent uses of |descriptor|'s
+        <a>permission state</a> with the same <a>current settings object</a>
+        must return the same value, unless the UA receives <a>new information
+        about the user's intent</a>.
+      </p>
 
-    <p class="issue" id="issue-current-entry-incumbent-or-relevant">
-      Safari is the only known UA that returns different results from this
-      algorithm for different settings objects with the same origin. We should
-      test which of the <a
-      href="https://html.spec.whatwg.org/multipage/webappapis.html#realms-settings-objects-global-objects">several
-      possible settings objects</a> it uses.
-    </p>
+      <p class="issue" id="issue-current-entry-incumbent-or-relevant">
+        Safari is the only known UA that returns different results from this
+        algorithm for different settings objects with the same origin. We should
+        test which of the <a
+        href="https://html.spec.whatwg.org/multipage/webappapis.html#realms-settings-objects-global-objects">several
+        possible settings objects</a> it uses.
+      </p>
 
-    <p>
-      Some <a>powerful features</a> have more information associated with them
-      than just a {{PermissionState}}. For example,
-      {{MediaDevices/getUserMedia()}} needs to determine <em>which</em> cameras
-      the user has granted the <a>current realm</a> permission to access. Each
-      of these features defines an <a>extra permission data type</a>, and then a
-      {{PermissionName}} |name|'s <dfn export>extra permission data</dfn> is the
-      instance of that type matching the UA's impression of the user's intent.
-      Subsequent uses of |name|'s <a>extra permission data</a> must return the
-      same value, unless the UA receives <a>new information about the user's
-      intent</a>.
-    </p>
+      <p>
```

Comment:
Sorry, I don't understand this comment :( 

In case I don't hear back from you in the next few hours; I'm about to shift focus to another project for 2 weeks, so could I kindly request that you fix this for me in the `editorial_fixes` branch? I've already fixed the `<var>`s (there are ~30 vars  that could maybe be converted also, but I guess we should do that in a new PR).